### PR TITLE
Changelog 1.15

### PR DESCRIPTION
**Pull request by MarianRaphael**
## Description

Changelog entries:

- E-Mail Notifications
- Blueprint Selection Update
- Device Groups 

- [ ] Screenshot for Device Groups

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/changelog/2023/12/blueprint-selection.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+date: 2023-12-21 10:00:00.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+We have updated the blueprint selection interface to enhance the user experience by offering a clearer and more comprehensive overview. This improvement aims to facilitate easier navigation and selection of blueprints. Additionally, we are excited to announce that our blueprint library will be expanding in the coming days, with the addition of new blueprints to FlowFuse.
+
+![](./images/blueprints-selection.png)
+
+For more detailed information about our blueprints and how to utilize them effectively, please refer to our dedicated [article](https://flowfuse.com/blog/2023/10/blueprints/).
```
Comment (ZJvandeWeg) [inline src/changelog/2023/12/blueprint-selection.md:12]:
```suggestion
For more detailed information about our blueprints and how to utilize them effectively, please refer to our dedicated [article](/blog/2023/10/blueprints/).
```

File: src/changelog/2023/12/blueprint-selection.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: Blueprint Selection Update
+date: 2023-12-21 10:00:00.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+We have updated the blueprint selection interface to enhance the user experience by offering a clearer and more comprehensive overview. This improvement aims to facilitate easier navigation and selection of blueprints. Additionally, we are excited to announce that our blueprint library will be expanding in the coming days, with the addition of new blueprints to FlowFuse.
+
+![](./images/blueprints-selection.png)
```
Comment (ZJvandeWeg) [inline src/changelog/2023/12/blueprint-selection.md:10]:
```suggestion
!["FlowFuse Blueprint Selection"](./images/blueprints-selection.png "FlowFuse Blueprint Selection")
```

File: src/changelog/2023/12/email-alerts.md
Comment (ZJvandeWeg) [inline src/changelog/2023/12/email-alerts.md:1]: can we make the title a bit more SEO friendly?

`email-alerting-node-red-crash.md`?

File: src/changelog/2023/12/email-alerts.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title: E-Mail Notifications
```
Comment (ZJvandeWeg) [inline src/changelog/2023/12/email-alerts.md:2]: When? Why? For who?

File: src/changelog/2023/12/email-alerts.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+date: 2023-12-21 12:00:00.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+
+The newly introduced feature offers email notifications tied to particular Auditlog events, available to all users with Team Tier access or higher. This feature is designed to facilitate immediate notification and action in response to significant occurrences, including incidents like a Node-RED Instance crash. For detailed information and guidance on this functionality, please refer to our [documentation](https://flowfuse.com/docs/user/instance-settings/#alerts).
```
Comment (ZJvandeWeg) [inline src/changelog/2023/12/email-alerts.md:9]:
```suggestion
The newly introduced feature offers email notifications tied to particular Audit log events, available to all users with Team Tier access or higher. This feature is designed to facilitate immediate notification and action in response to significant occurrences, including incidents like a Node-RED Instance crash. For detailed information and guidance on this functionality, please refer to our [documentation](/docs/user/instance-settings/#alerts).
```

> including incidents like a Node-RED Instance crash. 

Great, but lets list all events if they're a handful :)

**Review by ZJvandeWeg — APPROVED** [review]

**Review by ZJvandeWeg — APPROVED** [review]

**Review by knolleary — APPROVED** [review]

**Pull request merged**